docs/DEVGUIDE.md

```diff
@@ -14,45 +19,98 @@ You will need to meet the following prerequisites:
 * [Helm](https://helm.sh) installed in your Kubernetes cluster,
   and the `helm` binary in your `PATH`
 * Cluster credentials in ./kubeconfig file
-* GNU Make
-* [git](https://git-scm.com)
-* Must be pre-authenticated to any Docker registry to which you intend to push
-  images
 
-## Getting Sources
+Additionally, if you intend to push Docker images to a registry:
+* Must be pre-authenticated to any Docker registry
```

Comment:
Teeny, tiny nit... if someone wants to push to _a_ registry, they don't need to be authenticated to _any_ registry... they need to be authenticated to the one(s) they intend to push to. Obviously, you know this, I'm just pointing out that this reads oddly.